### DISCUSSION TITLE
5-1. 스멀스멀 피어나는 이직 욕구

### DISCUSSION THREAD

**Discussion by imakerjun** [ssac-dev/yozm#15] [5장 다른 무대에도 서보고 싶어]
# 5-1. 스멀스멀 피어나는 이직 욕구

## 1. 회사에서 이루고 싶은 나의 욕구 리스트 세 가지 작성하기
회사에 다니다 보면 이직 욕구가 스멀스멀 생기는 경우가 있습니다. 이럴 때일수록 나의 이직 욕구에 대한 ‘진단’과 ‘피드백’을 실시해야 합니다. 진단은 회사에 대한 아쉬운 부분보다 먼저 내가 원하는 것이 무엇인지 어떤 환경에서 일을 잘할 수 있는지 되돌아보는 것입니다. 무엇이 진짜 나에게 필요한지, 단순한 불편함 때문에 이직 생각이 드는지, 아니면 환경적인 개선이 필요한 것인지 체크해보면 해야 할 행동이 더 명확해집니다. 이를 위해 평소 내가 원하는 것과 일을 잘할 수 있는 환경에 대해 구체적인 리스트로 작성해두면 도움이 됩니다. 그리고 그 리스트를 통해 현재 환경에서 개선할 수 있는 것인지 아니면 다른 환경을 개척해야 하는 것인지 피드백을 받을 수 있습니다.

## 2. 내가 일을 잘할 수 있는 환경 요소 세 가지 작성하기
대부분의 사람들은 자신이 현재 놓여 있는 상황 전반을 객관적으로 이해하려고 하기보다 지금 처해 있는 문제에 집중하려는 경향이 있습니다. 이를 터널 비전(tunnel vision)이라고 하는데 터널 속으로 들어갔을 때 터널 안만 보이고 밖은 보이지 않는 것처럼 하나에 너무 몰입하여 주변 상황을 객관적으로 바라보지 못하는 상황을 의미합니다. 보통 힘든 환경이라고 하면 한 가지 어려움보다 여러 가지 상황이 복합적으로 연결되어 있는 경우가 많습니다. 그래서 때로는 한 발짝 떨어져서 봐야 이해되기도 합니다. 이를 위해 평소에 내가 어떤 환경에서 일을 잘하는지 기록해두면 어려운 상황에서도 넓은 시야로 문제를 바라볼 수 있습니다. 나중에 이직할 회사의 문화와 환경을 찾아볼 때도 이 리스트를 보면서 나에게 더 잘 맞는 환경으로 만들 수 있는 곳인지 판단할 수 있습니다. 이 리스트도 최대한 구체적, 능동적 문장으로 작성하는 것이 좋습니다.

## 3. 내가 이루고 싶은 일 중 현재 환경에서 시도할 수 있는 것 세 가지 작성하기
내가 이루고 싶은 것은 무엇인지, 내가 원하는 방향으로 나아가기 위해 가장 효율적으로 일할 수 있는 환경이 어떤 곳인지 파악했다면 현재 환경이이나 위치가 어떤지 파악해볼 차례입니다. 사실 이직에 대한 욕망이 피어오를 때 내가 있는 환경에서 약간 다른 시도를 해보면 이직보다 훨씬 더 쉽게 의미 있는 결과를 만들어낼 수도 있습니다. 그래서 욕망이 스멀스멀 피어오를 때일수록 내가 원하는 것이 무엇인지 확인해보는 것이 좋습니다. 나에게는 기술적인 도전, 인간관계, 사수의 유무, 근무 형태, 커리어 성장 등 다양한 욕구가 있을 수 있습니다. 이 욕구의 우선순위를 먼저 매겨보세요. 그리고 그 우선순위를 현재 상황에서 왜 달성하기 어려운지, 내가 해볼 수 있는 다른 시도는 없는지 고민해보는 것입니다.

**Comment by yeslkoh** [comment]
### 1. 입사 전 혹은 직후에 현재 회사에서 이루고 싶은 나의 욕구 리스트 세 가지 작성하기

> 1. 개발하는 서비스에 좀 더 집중을 해왔다면, 앞으로는 프로덕트를 소비하는 시장에 대한 이해도를 높이고 싶다.
> 2. 담당 역할의 개발 영역 이상의, 업계 전반에 대한 이해도를 높이고 싶다. 
> 3. 성장의 문화를 셋팅하고 장려하는 역할을 더욱 적극적으로 하고 싶다.

###  2. 내가 일을 잘할 수 있는 환경 요소 세 가지 작성하기
> 1. 같이 일하는 동료들 간의 신뢰가 있는 곳. (어떠한 질문을 해도 괜찮다는 심리적 안전감이 있는)
> 3. 일 외의 영역에서도 나를 자유로이 표현할 수 있는 곳.
> 4. 내가 닮고 싶은 부분을 가진 사람이 있는, 롤모델이 있는 곳.

### 3.  현재 환경에서 시도해볼 수 있는 것 세 가지 작성하기

> 1. 회사 사업 전반 프로세스 파악하기 - monetization팀과의 미팅 후 팀의 프로세스 파악
> 2. 회사 사업 전반 프로세스 파악하기 - monetization팀과 협업 방향 논의
> 3. 각 팀의 프로젝트 진행상황 공유하는 토크 자리 마련
